autogtp/main.cpp

```diff
@@ -57,7 +57,7 @@ int main(int argc, char *argv[]) {
     QCommandLineOption gamesNumOption(
         {"g", "gamesNum"},
               "Play 'gamesNum' games on one GPU at the same time.",
-              "num", "1");
+              "num", QString::number((QThread::idealThreadCount() + 1) / 2));
```

Comment:
This appears to be a merge error.

Reply:
Right. So I should leave it as 1? I know we have discussion that we should use 1 instead, but I thought the later is more flexible and we can change `idealThreadCount` instead.

Reply:
Reverted again.